### DISCUSSION TITLE
NextJS Build Error classNames

### DISCUSSION THREAD

**Discussion by tonyr1971** [recharts/recharts#2835] [Q&A]
Hi,

When building in Nextjs V12( npm run build ) get the following error, I have tried installing the type with npm i --save-dev @types/classnames but still get the error

Type error: Could not find a declaration file for module 'classnames'. 'C:/Dev/node_modules/classnames/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/classnames` if it exists or add a new declaration (.d.ts) file containing `declare module 'classnames';`

  4 | import _ from 'lodash';
  5 | import React, { PureComponent, CSSProperties, ReactNode } from 'react';
> 6 | import classNames from 'classnames';
    |                        ^

Thank you for your help

**Comment by lamhungypl** [comment]
It seems not related to chart lib, have you updated your dependencies to the latest major version?

or please provide a quick sandbox reproduction.
